app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragmentTest.kt

```diff
@@ -167,22 +165,6 @@ class MetadataEditingFragmentTest {
         )
     }
 
-    @Ignore("will be fixed later")
-    @Test
-    fun clickOnInfoButtonWorks() {
```

Comment:
There is no info button on this view anyway.

Reply:
#235

Reply:
ok